build.gradle

```diff
@@ -209,7 +215,10 @@ dependencies {
   testCompile group: 'org.junit.platform', name: 'junit-platform-commons', version: versions.junitPlatform
   testCompile group: 'org.junit.platform', name: 'junit-platform-engine', version: versions.junitPlatform
 
-  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test'
+  testCompile group: 'org.springframework.boot', name: 'spring-boot-starter-test', withoutJunit4
+  testCompile group: 'com.github.tomakehurst', name: 'wiremock-jre8', version: '2.25.0', withoutJunit4
+
+  integrationTestCompile group: 'org.springframework.cloud', name: 'spring-cloud-contract-wiremock', version: '2.1.3.RELEASE', withoutMockitoStandAlone
```

Comment:
this will make 2 types of wiremock present 🤔

Reply:
we exclude the one comes from spring-cloud-contract-wiremock and it should 1 in classpath

Reply:
ah there's `standalone` and `jre8` dependencies

Reply:
next PR I will make it integration scope